src/test/java/com/bloxbean/cardano/client/backend/api/helper/UtxoTransactionBuilderTest.java

```diff
@@ -365,13 +366,13 @@ public void testBuildTransactionWithMultiAssetAndNotSufficientLovelaceWillReturn
         assertThat(transaction.getBody().getInputs().get(1).getTransactionId(), is("aaaaaa341088ca1c0ed2384a3139d34a1de4b31ef6c9cd3ac0c4eb55108fdf85"));
         assertThat(transaction.getBody().getInputs().get(2).getTransactionId(), is("bbbbbb341088ca1c0ed2384a3139d34a1de4b31ef6c9cd3ac0c4eb55108fdf85"));
 
-        assertThat(transaction.getBody().getOutputs(), hasSize(3));
+        assertThat(transaction.getBody().getOutputs(), hasSize(2));
```

Comment:
@satran004 here the result is well different, my changes reduce the number of created outputs, hence one less output. 

same as below change.

Reply:
Cool. Hence less fee :)